README.md

```diff
@@ -255,6 +258,26 @@ inner A
 Since the first listener uses `(stop-immediate-propagation)`, the second
 listener is not called. The `.outer` listener is also not called.
 
+#### `(target-value)`
+
+Accesses [`event.target.value`][e-target] and passes it on.
+
+```hbs
+<input type="text" {{on "input" (target-value this.processValue)}}>
+```
```

Comment:
Maybe it would be more useful, if this helper takes a key path param similar to how `value` of `{{action}}` did. This would allow the user to pluck off a different value, like `target.checked`, which you'd be interested in instead when dealing with `<input type="checkbox">`.

Reply:
The original idea was to be param-free. But here are ideas:

- `(target-value this.foo)` -> as is
- `(target-value this.foo 'checked')` -> for checkbox (it's not really `target-value` anymore)
- `(target-check this.foo)` -> I prefer this for the time being.

WDYT ?